install/installing-on-linux.md

```diff
@@ -69,10 +69,10 @@ Now create a `.env` config file in the base of repository. Make sure that the da
 
 ```
 DB_HOST=localhost
-DB_NAME=username
-DB_PASS=password
+DB_NAME=ushahidi
 DB_TYPE=MySQLi
-DB_USER=ushahidi
+DB_USER=username
```

Comment:
@cmacdonell Do you think it makes more sense to just show `username` here? or use `ushahidi` for both? 
the GRANT command above shows creating both a DB and user named `ushahidi`.
My feeling is that we should either match the credentials we used above, or use a placeholder for db too `DB_NAME=database`